app/models/manageiq/providers/embedded_ansible/seeding.rb

```diff
@@ -20,7 +20,13 @@ def seed
         :resource => manager
       )
 
-      Ansible::Content.consolidate_plugin_playbooks
+      # In production mode, the RPM build takes cares of consolidating all of
+      #   the plugin content, which will not change, so this is unnecessary.
+      # In development mode, changes to content will be reconsolidated on every
+      #   seed for convenience.
+      unless Rails.env.production?
+        Ansible::Content.consolidate_plugin_content
```

Comment:
still feel this would be better suited for /bin/update or /bin/setup

but I'm ok with this

Reply:
yeah, I didn't want to rock this boat too much, so I left it...we can definitely debate dropping it from here, but I'd rather do that in a follow up.

I really don't understand the use case of why it's here in seed in the first place.  From what I can tell it's for developers modifying things, but then, I don't see why they can't just call evm:ansible_runner:seed themselves.